grails-app/views/picklist/_printPage.gsp

```diff
@@ -133,39 +133,7 @@
                             <!-- Checked by -->
                         </td>
                         <td class="middle">
-                            <g:if test="${requisitionItem?.parentRequisitionItem?.cancelReasonCode}">
```

Comment:
Do we need this td column, if it is empty?

Reply:
We need it to have borders so it can be filled after printing